tests/test_pipeline.py

```
@@ -0,0 +1,321 @@
+        _setup_model(p, ["skill_a:my.intent"], [0.9])
+        # no intents registered and not a special label → mask is all-False → empty
+        results = list(p._match("something"))
+        self.assertEqual(results, [])
+
+    def test_ocp_special_case_bypasses_intents_check(self):
+        # ocp:play must match even with no Adapt/Padatious intents registered
+        p = _make_pipeline(intents=[], renormalize=False)
+        _setup_model(p, ["ocp:play"], [0.95])
+        results = list(p._match("play some music"))
+        self.assertEqual(len(results), 1)
+        skill_id, label, prob = results[0]
```

Ruff (0.15.2)

[warning] Unpacked variable `prob` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)